Source/WebCore/css/CSSPrimitiveValueMappings.h

```diff
@@ -107,12 +186,11 @@ DEFINE_TO_FROM_CSS_VALUE_ID_FUNCTIONS
 #undef TYPE
 #undef FOR_EACH
 
-template<> inline CSSPrimitiveValue::operator ColumnSpan() const
+template<> inline ImplicitValueMapper::operator ColumnSpan() const
 {
-    // Map 1 to none for compatibility reasons.
```

Comment:
Can we keep this comment?

Reply:
I would be wiling to keep it, but hear my out about why I don’t think it’s a good comment:

1) The comment doesn’t say enough. Compatibility with what? There’s no reference here to code, a specification, a particular website, or even a test.
2) The comment is in the wrong place. Mapping the 1 value when converting the enumeration would be only part of what’s needed, because we also need to at least parse that value. The whole point of these mapping functions is that they only work when the value is already known to be one of the correct ones, and the code that guarantees that is the CSS parser, and to a lesser extent the CSS computed style extractor.

On the other hand, I think the real issue is that this is dead code. I don’t think we do map 1 to none for compatibility reasons! We certainly don’t do it in the parsing code for `column-span` nor in the computed style extraction. I’ll leave this poor comment in for now at your request, and then I’ll return and prove that this is not needed and delete it.

Reply:
Yes, removing the code + the comment entirely in a separate commit seems better.

Removing just the comment leaves the risk that this code is never actually removed if it is deprecated.